app/src/main/java/com/sun_asterisk/moviedb_50/data/source/local/DatabaseHelper.kt

```diff
@@ -0,0 +1,33 @@
+package com.sun_asterisk.moviedb_50.data.source.local
+
+import android.content.Context
+import android.database.sqlite.SQLiteDatabase
+import android.database.sqlite.SQLiteOpenHelper
+import com.sun_asterisk.moviedb_50.data.model.Favorite
+import com.sun_asterisk.moviedb_50.utils.Constant
+
+class DatabaseHelper private constructor(context: Context?) : SQLiteOpenHelper(

```

Comment:
sao context lại có thể null được 
```suggestion
class DatabaseHelper private constructor(context: Context) : SQLiteOpenHelper(

```